widgetry/src/shaders/vertex_140.glsl

```diff
@@ -25,9 +25,10 @@ void main() {
     // Translate that to normalized device coordinates (NDC)
     float x = (screen_x / window[0] * 2.0) - 1.0;
     float y = (screen_y / window[1] * 2.0) - 1.0;
+    float z = position[2] + window[2];
```

Comment:
Ah, this is the part that was confusing me. Makes perfect sense -- lots of times, we want to conveniently set a base Z value in the uniforms, but now we're adding the ability to tune this per polygon. Could we add a comment to that effect in `drawing.rs` by the Z value definitions, and mention keeping the polygon Z value under 1.0 (for the reasons you had in the PR description)?